### DISCUSSION TITLE
Wordpress Plugin UI Customisation

### DISCUSSION THREAD

**Discussion by danielemanca1983** [bullhorn/career-portal#535] [Q&A]
Hi folks,

I am working on a client's Wordpress site, on which they have used Bullhorn Wordpress Plugin. 

They would like to update the UI of the careers page to follow brand colours etc, however I have been unable to do so by adding custom CSS.

My question would then be: are there any ways to customise the look and feel of the UI, other than editing the core CSS file of the plugin?

**Comment by coLlenz** [comment]
Yes, you only customize through the core css, the css which has the minified one.

**Comment by pbearne** [comment]
Hi Daniel

It sounds like you should look at the Matador Jobs WordPress plugin https://matadorjobs.com/

As the brand works out of the box inheriting from the theme as the Jobs are just CPT in WordPress

reach out for demo

Paul

**Reply by coLlenz** [reply]
In Matador Jobs, Is there an option in your plugin not to require the Resume/CV?

**Reply by pbearne** [reply]
Yes
We have done it for many sites
reach out for a demo

**Comment by soireemarketing** [comment]
Hi, customising the OSCP is technical but achievable. This is something i'm able to help you with - feel free to look at my website and reach out if you want your OSCP to be customised.

**Reply by pbearne** [reply]
@soireemarketing I would love to give you a demo of Matador Jobs so you can recommend us if an OSCP client needs more customizing than doable with OSCP

**Reply by danielemanca1983** [reply]
Thanks but I were able to customise Bullhorn without the need to switch to any other plugin/solutions, I know it's not the best way of doing it, but I simply edited the CSS file in the root of the plugin.

**Reply by pbearne** [reply]
Watch if/when a new version is released
Do you need Google to index the jobs?
Point this tool at your job board https://developers.google.com/search/docs/appearance/structured-data to check that the Job Posting meta is working https://developers.google.com/search/docs/appearance/structured-data/job-posting
